packages/context/src/__tests__/acceptance/interceptor.acceptance.ts

```diff
@@ -239,6 +240,75 @@ describe('Interceptor', () => {
     });
   });
 
+  context('proxy with interceptors', () => {
+    it('invokes async interceptors on an async method', async () => {
+      const proxy = createProxyWithInterceptors(
+        controllerWithClassInterceptors,
+        ctx,
+      );
+      const msg = await proxy.greet('John');
```

Comment:
Please move the controller class & method definition to individual tests, as I explained in one of my earlier comments, so that reader of this code can understand what scenario is being tested, without having to jump to the definition of `controllerWithClassInterceptors`. The same comment applies to other tests added by this commit too.